packages/dds/cell/src/cell.ts

```diff
@@ -287,7 +287,7 @@ export class SharedCell<T extends Serializable = any> extends SharedObject<IShar
             if (local) {
                 const messageIdReceived = localOpMetadata as number;
                 assert(messageIdReceived !== undefined && messageIdReceived <= this.messageId,
-                    "messageId is incorrect from from the local client's ACK");
+                    "s_00" /* messageId is incorrect from from the local client's ACK */);
```

Comment:
Why use string for short code? Can the short code be just numbers?

Reply:
From a bundle size perspective, I suspect base64 encoded numbers would be the biggest bang for our buck

Reply:
Yeah, on the flip side, I am thinking of runtime memory size would be better with numbers.

Reply:
in terms of bundle size strings have a higher information density, could do hex numbers i guess which is a bit better

Reply:
i wasn't really concerned with in memory size

Reply:
currently they are radix 36 number strings, with supports 1296 unique codes for 2 characters, if and when we hit 3 chars, it goes up to 46656. we are not even very close the to two char limit (394), but if we ever cross that we should never hit the 3 char limit 46656. the current implementation  is a good balance between information density and complexity, `num.tostring(36)`

Reply:
if we care about memory size we can move to hex, we'll need 3 chars, which is probably fine, and but will also require code change